internal/pkg/service/buffer/worker/service/file_test.go

```diff
@@ -0,0 +1,109 @@
+package service_test
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/c2h5oh/datasize"
+	"github.com/keboola/go-utils/pkg/wildcards"
+	gonanoid "github.com/matoous/go-nanoid/v2"
+	"github.com/stretchr/testify/assert"
+
+	"github.com/keboola/keboola-as-code/internal/pkg/service/buffer/api/gen/buffer"
+	service2 "github.com/keboola/keboola-as-code/internal/pkg/service/buffer/api/service"
+	bufferDependencies "github.com/keboola/keboola-as-code/internal/pkg/service/buffer/dependencies"
+	"github.com/keboola/keboola-as-code/internal/pkg/service/buffer/store/model"
+	"github.com/keboola/keboola-as-code/internal/pkg/service/buffer/worker/service"
+	"github.com/keboola/keboola-as-code/internal/pkg/service/common/dependencies"
+	"github.com/keboola/keboola-as-code/internal/pkg/utils/errors"
+	"github.com/keboola/keboola-as-code/internal/pkg/utils/etcdhelper"
+	"github.com/keboola/keboola-as-code/internal/pkg/utils/strhelper"
+	"github.com/keboola/keboola-as-code/internal/pkg/utils/testhelper"
+	"github.com/keboola/keboola-as-code/internal/pkg/utils/testproject"
+)
+
+func TestUploadAndImportE2E(t *testing.T) {
+	t.Parallel()
+
```

Comment:
A tu je E2E vsetkeho, od vytvorenia records cez API az po import :tada: 

Bezi to sice `30s`, lebo sa musi vytvorit bucket, table, file, token, ale je to celkom dobre citatelne.
Mozno este upravim aj ostetne worker testy takto.

Reply:
super 🚀

Reply:
Ale file ve Storagi teda nekontroluješ?

Reply:
Chcel som kontrolovat rovno obsah tabulky (aby to bolo fakt E2E), ale nemame v `go-client` API call na table preview, ... chcem to prebrat zajtra na stand-upe, ci nam to dava zmysel pridat.

S kontrolovanim (stiahnutim) slicovaneho suboru je problem, ak si spominas.

Mame to sice tu: 
https://github.com/keboola/keboola-as-code/blob/4d4e8c153be9bdf35321f8b30de7f9ca715ed6d0/internal/pkg/service/buffer/worker/service/slice_upload_test.go#L146-L152

Ale len per Azure, z S3 sa neda stiahnut.

-----

Co mozem pridat, je skontrolovanie poctu riadkov v tabulke, `GetTableRequest` mame, to rano doplnim.

Reply:
Doplnene: https://github.com/keboola/keboola-as-code/pull/1153/commits/ed39e56843ca935fa102e8d4e7fca47e27d72a55, https://github.com/keboola/keboola-as-code/pull/1153/commits/cf0c42d9fc3be1a9c4816d89d6313299619e580e